src/Xamarin.Android.Build.Tasks/Tasks/AndroidApkSigner.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Xamarin.Android.Tools;
+
+namespace Xamarin.Android.Tasks
+{
+	public class AndroidApkSigner : AndroidToolTask
+	{
+		[Required]
+		public string ApkToSign { get; set; }
+
+		[Required]
+		public ITaskItem ManifestFile { get; set; }
+
+		[Required]
+		public string KeyStore { get; set; }
+
+		[Required]
+		public string KeyAlias { get; set; }
+
+		[Required]
+		public string KeyPass { get; set; }
+
+		[Required]
+		public string StorePass { get; set; }
+
+		public string AdditionalArguments { get; set; }
+
+		protected override string GenerateCommandLineCommands ()
+		{
+			var cmd = new CommandLineBuilder ();
+
+			var manifest = AndroidAppManifest.Load (ManifestFile.ItemSpec, MonoAndroidHelper.SupportedVersions);
+			int minSdk = MonoAndroidHelper.SupportedVersions.MaxStableVersion.ApiLevel;
```

Comment:
This seems at odds with e.g. `ManifestDocument.GetMinimumSdk()`, which defaults to 11 if it's not specified.

Reply:
this will be handled and updated via xamarin/xamarin-android-tools#11